docs/Developers/Writing_Good_Commits.rst

```diff
@@ -26,6 +26,9 @@ Example of a good commit:
 - `Fixes https://github.com/coala/coala/issues/5861`: Mention the URL
    of the issue it closes or fixes.
 
+Use 'Fixes' when you fixed a bug and 'Closes' when your work is 
+related to a new feature.
```

Comment:
This needs to be more specific. You should clarify that fixes is for *any* unintended behavior even for developers. (my interpretation of the discussion is that any unintended behavior is a bug whether that is the user or developer).